tests/integration/service_test.py

```diff
@@ -245,72 +249,6 @@ def test_create_container_with_volumes_from(self):
         self.assertIn(volume_container_2.id,
                       host_container.get('HostConfig.VolumesFrom'))
 
-    def test_converge(self):
-        service = self.create_service(
-            'db',
-            environment={'FOO': '1'},
-            volumes=['/etc'],
-            entrypoint=['top'],
-            command=['-d', '1']
-        )
-        old_container = service.create_container()
-        self.assertEqual(old_container.get('Config.Entrypoint'), ['top'])
-        self.assertEqual(old_container.get('Config.Cmd'), ['-d', '1'])
-        self.assertIn('FOO=1', old_container.get('Config.Env'))
-        self.assertEqual(old_container.name, 'composetest_db_1')
-        service.start_container(old_container)
-        old_container.inspect()  # reload volume data
-        volume_path = old_container.get('Volumes')['/etc']
-
-        num_containers_before = len(self.client.containers(all=True))
-
-        service.options['environment']['FOO'] = '2'
-        new_container = service.converge()[0]
-
-        self.assertEqual(new_container.get('Config.Entrypoint'), ['top'])
-        self.assertEqual(new_container.get('Config.Cmd'), ['-d', '1'])
-        self.assertIn('FOO=2', new_container.get('Config.Env'))
-        self.assertEqual(new_container.name, 'composetest_db_1')
-        self.assertEqual(new_container.get('Volumes')['/etc'], volume_path)
-        self.assertIn(
-            'affinity:container==%s' % old_container.id,
-            new_container.get('Config.Env'))
-
-        self.assertEqual(len(self.client.containers(all=True)), num_containers_before)
-        self.assertNotEqual(old_container.id, new_container.id)
-        self.assertRaises(APIError,
-                          self.client.inspect_container,
-                          old_container.id)
-
-    def test_converge_when_containers_are_stopped(self):
-        service = self.create_service(
-            'db',
-            environment={'FOO': '1'},
-            volumes=['/var/db'],
-            entrypoint=['top'],
-            command=['-d', '1']
-        )
-        service.create_container()
-        self.assertEqual(len(service.containers(stopped=True)), 1)
-        service.converge()
-        self.assertEqual(len(service.containers(stopped=True)), 1)
-
-    def test_converge_with_image_declared_volume(self):
-        service = Service(
-            project='composetest',
-            name='db',
-            client=self.client,
-            build='tests/fixtures/dockerfile-with-volume',
-        )
-
-        old_container = create_and_start_container(service)
-        self.assertEqual(old_container.get('Volumes').keys(), ['/data'])
-        volume_path = old_container.get('Volumes')['/data']
-
-        new_container = service.converge()[0]
-        self.assertEqual(new_container.get('Volumes').keys(), ['/data'])
-        self.assertEqual(new_container.get('Volumes')['/data'], volume_path)
-
```

Comment:
Isn't this just testing `converge()` which is a test-only helper?

I'll have to verify that there are tests for `execute_convergence_plan()`